portfolio/src/main/webapp/script.js

```diff
@@ -38,3 +38,33 @@ function addRandomStock() {
   const stockContainer = document.getElementById('stock-container');
   stockContainer.innerText = stock;
 }
+
+// Fetches comments from the server and adds them to the DOM.
+function LoadComments(){
+  var max=getMax();
+  fetch('/data?user-choice='+max).then(response => response.json()).then((comments) => {
```

Comment:
nit: please add spaces before and after the `=` here and later in the function

Reply:
Addressed in latest commit.